.github/workflows/release.yaml

```diff
@@ -10,6 +10,8 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      - name: Export RELEASE_TAG var
+        run:  echo "RELEASE_TAG=${GITHUB_REF:10}" >> $GITHUB_ENV
```

Comment:
Why should we hard code it to 10 here?

Reply:
`:10 `strips first ten characters from `GITHUB_REF` var. In other words, instead of  `refs/tags/vx.y.z`  we will get `vx.y.z`